index.html

```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
```

Comment:
Looks weird saying **number 1**, either have it **no. 1** or **number one**. Something doesn't seem right with this